ovirt-engine.spec.in

```diff
@@ -295,6 +295,7 @@ Requires:	rhvm = %{version}-%{release}
 # bump the requirement (otherwise we would need to pass wildfly,
 # wildfly-overlay and ovirt-engine through the change queue at once)
 Requires:	ovirt-engine-wildfly-overlay >= 24
+Requires:	ovirt-engine-keycloak
```

Comment:
Why do we need this? Shouldn't ovirt-engine-keycloak package be added into version locked packages and getting installed/upgraded only as a part of ovirt-engine-keycloak-setup similar to DWH?

https://github.com/oVirt/ovirt-dwh/blob/master/packaging/setup/plugins/ovirt-engine-setup/ovirt-engine-dwh/distro-rpm/packages.py

Reply:
That sounds like a good idea. I have not seen that code earlier. I'll make necessary updates.